opdracht1.js

```diff
@@ -0,0 +1,33 @@
+const tvNames = inventory.map ((tv) =>{
+    return tv.type;
+})
+
+console.log("1A; Tv types", tvNames);
+
+const soldOut = inventory.filter((tv) =>{
+    return tv.originalStock === tv.sold;
+})
+
+console.log("1B; Sold out ", soldOut);
+
+const ambiTv = inventory.filter((tv) =>{
+    return tv.options.ambiLight === true;
+})
```

Comment:
Omdat `ambiLight` van zichzelf al een boolean is hoeft er geen expliciete check gedaan te worden
```suggestion
const ambiTv = inventory.filter(tv => tv.options.ambiLight);
```